docs/advanced_configuration.md

```diff
@@ -201,6 +201,8 @@ All interactions with the RUM SDK (view tracking, actions, resources tracing, an
 
 Logs are not recorded and calling a `DdLogs` method before the actual initialization might break logging.
 
+Refer to the [`example app`](https://github.com/DataDog/dd-sdk-reactnative-examples/tree/main/rum-react-navigation-async) to see an asynchronous implementation of Datadog with React navigation.
```

Comment:
I'm not sure the mention to React Navigation is very important in this example. By mentioning it we put less focus on the asynchronous implementation that is the important term, and we probably don't want users to find this sentence when they look for React Navigation.
So I'd be in favor of removing the "with React Navigation", what do you think?

My comment also applies for other examples except the ones in the React Navigation sections

Reply:
If we think about it that way, I agree. Especially if we link to it in a way to check after errors